docs/docusaurus/docs/cloud/quickstarts/python_api_quickstart.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: 'Quickstart for GX Cloud and Python scripts'
+title: 'Quickstart for GX Cloud and Python scripts'
+id: python_quickstart
+description: Connect to a GX Cloud account from a Python script.
```

Comment:
```suggestion
description: Use Python to set up GX Cloud.
```

Reply:
The original description is more accurate: They aren't actually setting up GX Cloud through Python -- they are still following the standard setup steps for getting an account; the only difference is that they are setting their environment variables outside of docker.  That still isn't doing any setup in Python, however.  Once they are in Python when they run `get_context()` it will detect those environment variables and initialize a GX Cloud Data Context, which will be connected to their GX Cloud account.